index.bs

```diff
@@ -546,28 +568,93 @@ canvas.height = Math.max(leftEye.renderHeight, rightEye.renderHeight);
 </div>
 
 
-## VRStageParameters ## {#interface-vrstageparameters}
+## VRCoordinateSystem ## {#interface-vrcoordinatesystem}
+
+The VRCoordinateSystem interface defines a cartesian coordinate system measured in meters.  {{VRPose}} and {{VRFrameData}} matrices are always expressed in the context of a VRCoordinateSystem.
+The coordinate system uses these axis definitions:
 
-The {{VRStageParameters}} interface represents the values describing the the stage/play area for devices that support room-scale experiences.
+* Positive X is to the right of the origin.
+* Positive Y is up from the origin.
+* Positive Z is behind the origin.
 
 <pre class="idl">
-interface VRStageParameters {
-  readonly attribute Float32Array sittingToStandingTransform;
+interface VRCoordinateSystem {
+  boolean getTransformTo(VRCoordinateSystem coordinateSystem, Float32Array transform);
+};
+</pre>
+
+### Attributes ### {#vrcoordinatesystem-attributes}
+
+<dfn method for="VRCoordinateSystem">getTransformTo()</dfn>
+Fills in the transform to the supplied {{VRCoordinateSystem}} as a 16-element array containing the components of a 4x4 affine transformation matrix in column-major order.  Returns true if a transform exists between the origins of the two {{VRCoordianteSystem}} objects at the current frame's timestamp.  Returns false if the two {{VRCoordinateSystem}}s cannot be related for any reason (e.g. positional tracking loss).
```

Comment:
at the end: "or if the provided `transform` array is not at least 16 elements in length."